android-emulator-plugin/src/main/java/com/quittle/androidemulator/task/InstallSdkDependenciesTask.java

```diff
@@ -14,7 +14,8 @@ public InstallSdkDependenciesTask(final EmulatorConfiguration emulatorConfigurat
         super(InstallSdkDependenciesTask.class, emulatorConfiguration);
 
         this.setExecutable(emulatorConfiguration.getSdkManager());
-        this.setArgs(Arrays.asList(buildSdkRootArgument(), "emulator", "cmdline-tools;latest", "platform-tools"));
+        this.args(Arrays.asList(buildSdkRootArgument(), "emulator", "cmdline-tools;latest", "platform-tools"));
+        this.args(emulatorConfiguration.getAdditionalSdkManagerArguments());
```

Comment:
> Do you think these args would make sense to additionally be added here?
Yes, it will definitely make sense. I missed that. How would you like to do that? You can reject this push and I will add args in the next one.

> One thing to keep in mind is that what you changed could be used on a really old version of the sdkmanager while I pointed you at the newest one, which we use to install the actual SDK packages.

I've checked args (--proxy*) for cmdline-tools;latest. It looks like they are still there. I don't see the difference.

Anyway, thanks a lot.

Reply:
> How would you like to do that?

I'm a big fan of having just one commit per change. Can you make the changes and squash them into one commit then force push to your branch?

Reply:
done.